From 0371944d43f913771d07be317edf5bec5f9aca0f Mon Sep 17 00:00:00 2001
From: Phantom X <PhantomX@users.noreply.github.com>
Date: Tue, 19 Jan 2021 08:02:01 -0300
Subject: [PATCH] Disable automatic autostart settings

This is not good when there is no manual configuration
---
 src/main-status-icon.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/main-status-icon.c b/src/main-status-icon.c
index ee1f095..e230dd3 100644
--- a/src/main-status-icon.c
+++ b/src/main-status-icon.c
@@ -313,7 +313,7 @@ static void
 __lambda9_ (void)
 {
 	XnpApplication* _tmp0_;
-	xfce_autostart_set ("xfce4-notes-autostart", "xfce4-notes", TRUE);
+	//xfce_autostart_set ("xfce4-notes-autostart", "xfce4-notes", TRUE);
 	_tmp0_ = application;
 	xnp_application_quit (_tmp0_);
 }
@@ -420,7 +420,7 @@ _vala_main (gchar** args,
 	g_signal_connect ((GApplication*) app, "activate", (GCallback) ___lambda11__g_application_activate, NULL);
 	g_set_application_name (_ ("Notes"));
 	build_plugin ();
-	xfce_autostart_set ("xfce4-notes-autostart", "xfce4-notes", FALSE);
+	//xfce_autostart_set ("xfce4-notes-autostart", "xfce4-notes", FALSE);
 	gtk_main ();
 	_g_object_unref0 (application);
 	application = NULL;
-- 
2.44.0

